resources/install.rb

```diff
@@ -60,7 +60,7 @@
   remote_file "apache #{new_resource.version} tarball" do
     source tarball_uri
     path "#{Chef::Config['file_cache_path']}/apache-tomcat-#{new_resource.version}.tar.gz"
-    verify { |file| validate_checksum(file) }
+    verify { |file| validate_checksum(file) } if new_resource.verify_checksum
```

Comment:
`...  if new_resource.verify_checksum.nil? ? !URI.parse(tarball_base_path).scheme == "file" : new_resource.verify_checksum`

the default value being true doesn't really make sense when the tarball_base_path is a file if its broken and is just going to throw an exception.